specification/purview/data-plane/Azure.Analytics.Purview.DataQuality/preview/2022-09-01-preview/purviewDataQuality.json

```diff
@@ -0,0 +1,3820 @@
+              }
+            }
+          },
+          "default": {
+            "description": "An error response received from the DataQuality Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "RuleTrendings_Get": {
+            "$ref": "./examples/RuleTrendings_Get.json"
+          }
+        }
+      }
+    },
+    "/accounts/{accountId}/assets/{assetId}/trends/rules": {
+      "post": {
+        "tags": [
+          "DataQuality Trending"
+        ],
+        "description": "Get Rules Trending By Rule Id.",
+        "operationId": "RuleTrendingsById_Get",
+        "consumes": [
+          "application/json-patch+json",
+          "application/json",
+          "text/json",
+          "application/*+json"
+        ],
+        "produces": [
+          "text/plain",
+          "application/json",
+          "text/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "accountId",
+            "description": "Get Account Id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "in": "path",
+            "name": "assetId",
+            "description": "Get Asset Id",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Get Rule Trending Query Payload",
+            "schema": {
+              "$ref": "#/definitions/RuleTrendingQueryPayload"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/RuleTrending"
+            }
+          },
+          "default": {
+            "description": "An error response received from the DataQuality Service",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "type": "string",
+                "description": "The error code"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "RuleTrendingsById_Get": {
+            "$ref": "./examples/RuleTrendingsById_Get.json"
+          }
+        }
+      }
+    },
+    "/accounts/{accountId}/validation": {
+      "post": {
+        "tags": [
+          "DataQuality Schema Validation"
+        ],
+        "description": "Schema Validating Request.",
+        "operationId": "ViewSchema_Validate",
+        "consumes": [
+          "application/json-patch+json",
```

Comment:
Why does this operation accept "application/json-patch+json"? What is it patching?